### PR TITLE
Update ea_apply_normalization_tofile.m

### PR DESCRIPTION
**Pull request by GGonEsc**
Fixing problems with SPM normalization needing to use and delete some temporal files.

### CODE REVIEW

File: ea_apply_normalization_tofile.m [hunk 1]
```diff
@@ -58,6 +58,7 @@ function ea_apply_normalization_tofile(options,from,to,directory,useinverse,inte
         ea_fsl_apply_normalization(options,from,to,useinverse,refim,'',interp);
 
     otherwise % SPM part here
+        tempdir = pwd;% ensure using a writable folder (not necessarily tempdir)
```
Comment (ningfei) [inline ea_apply_normalization_tofile.m]: Do you mean that the default tempdir can be not writable in some cases?

Comment (ningfei) [reply ea_apply_normalization_tofile.m]: this line can be dangerous since the original .nii.gz file might be deleted if it is in the current folder.

**Comment by GGonEsc** [comment]
Yes, I meant  the default tempdir is not writable in many cases (e.g. when
working in a server or cluster).
I do agree that this may be dangerous for the original files, but the
"from" file has a temporary/general name, which should make this deletion
save, right?
A different safety solution would be to check that the user is not working
inside the patient directory if that's the case just cd to an upper level
;) so pwd is always writable
Best,
Gabriel

**Comment by ningfei** [comment]
I see! But the "from" file here has its original name. It's not a copy with UUID in the name. `tempdir` is used in multiple places in LeadDBS. I think we had similar issue before when we ran batch processing on a cluster. A even more simpler solution would be specifying the `TMPDIR` for the job submitted or set `tempdir` in matlab before the calculation starts.

**Pull request closed without merging**